### PR TITLE
tests: fix flaky isPositionFixed check in oopif-scripts

### PR DESCRIPTION
**Pull request by connorjclark**
ref #13847

### CODE REVIEW

File: lighthouse-cli/test/fixtures/oopif-scripts.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html>
-  <body>
+  <body style="height: 200vh">
     <h1>Hello frames</h1>
     <iframe id="iframe-1" src="http://localhost:10200/oopif-simple-page.html" style="position: fixed"></iframe>
-    <iframe id="iframe-2" src="http://localhost:10503/oopif-simple-page.html" style="position: fixed"></iframe>
+    <iframe id="iframe-2" src="http://localhost:10503/oopif-simple-page.html" style="position: fixed; top: 50vh"></iframe>
```
Comment (connorjclark) [inline lighthouse-cli/test/fixtures/oopif-scripts.html:6]: not necessary, but it was weird how these two elements overlapped

**Review by brendankenny — APPROVED** [review]
LGTM!

**Review by adamraine — APPROVED** [review]

**Comment by brendankenny** [comment]
unfortunately still flaky :(

**Comment by connorjclark** [comment]
> unfortunately still flaky :(

(spoke already offline) Sorry, this PR is not addressing the real problem in CI, just a weird failure that only happens locally.

**Pull request merged**